java/opendcs/src/main/java/decodes/tsdb/algo/ScalerAdder.java

```diff
@@ -109,7 +109,9 @@ protected void doAWTimeSlice()
 			pr.missingAction != MissingAction.IGNORE)
 		 || ((pr = getParmRef(t = "input8")) != null && isAssigned(t) && isMissing(input8) && 
 			pr.missingAction != MissingAction.IGNORE)
-		 || ((pr = getParmRef(t = "input0")) != null && isAssigned(t) && isMissing(input9) && 
+		 || ((pr = getParmRef(t = "input9")) != null && isAssigned(t) && isMissing(input9) &&
```

Comment:
good catch, I'm going to yank this out into a separate PR though, scalaradder30 was copy paste and problem also has this mistake and a separate commit will be easier to port to the 7.0 branch.